### PR TITLE
Delay/throttle fix when search resumes, preventing burst of search requests

### PR DESCRIPTION
**Pull request by taihun81**
## Description
Fix for burst of search requests after search resumes, resulting in soft bans.

## Motivation and Context
I have added interval delays to when search queue gets filled up, so that each search thread can only initiate the search at given fixed interval.  That way even if the search resumes after extended period of pause, the search threads won't burst out the request all at once.  **Please comment if the delay interval handled in search queue is a good idea or not**

Spawn point search thread has been left untouched.

## How Has This Been Tested?
Ran, tested repeats of pause and resume for various intervals.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: pogom/search.py [hunk 1]
```diff
@@ -295,6 +307,8 @@ def search_worker_thread(args, account, search_items_queue, parse_lock, encrypti
             if args.proxy:
                 api.set_proxy({'http': args.proxy, 'https': args.proxy})
 
+            api.activate_signature(encryption_lib_path)
```
Comment (xrobau) [inline pogom/search.py]: I think this is handled elsewhere now?

Comment (Cy4n1d3) [inline pogom/search.py]: It's also missing the proxy authentification thingy.

Comment (taihun81) [inline pogom/search.py]: That diff is just pulling API activation out of the loop as it just need to be called once.  Everything else is untouched,

**Comment by Goldvivi1** [comment]
if it helps prevent my account from getting softban everytime i change location, im in on this

**Comment by lduda007** [comment]
Works great for me. Should be merged into develop branch imo.

**Comment by Cy4n1d3** [comment]
So far no problems on my side, thanks :)

**Comment by Racle** [comment]
Works like a charm. Should be merged.

**Comment by FlarumHostingCo** [comment]
I get an error  trying to start run.py 

NameError: global name 'random' is not defined

**Comment by taihun81** [comment]
Sounds like you didn't import random module.  Suspecting you are behind #492 ?

**Comment by FlarumHostingCo** [comment]
Thanks!  Yeah I'm still using old AAHAAAA repo and cherry picking my needs.. Ok I'll try that thanks!

**Pull request closed without merging**